src/Connections/FeedConnection.ts

```diff
@@ -194,17 +197,17 @@ export class FeedConnection extends BaseConnection implements IConnection {
     }
 
     public async handleFeedEntry(entry: FeedEntry): Promise<void> {
-        // We will need to tidy this up.
-        if (this.state.template?.match(/\$SUMMARY\b/) && entry.summary) {
-            // This might be massive and cause us to fail to send the message
-            // so confine to a maximum size.
+        // This might be massive and cause us to fail to send the message
+        // so confine to a maximum size.
+
+        if (entry.summary) {
```

Comment:
This was a mistake we missed last time around. We should actually attempt to trim this down regardless of the template, as even the embedded data content in the event might be too large.

We've seen a few of these in the wild from blogs that publish the whole article in `<summary>`.